### PR TITLE
Add redmine_gtt plugin

### PR DESCRIPTION
**Pull request by sanak**
Supports #1.

Changes proposed in this pull request:
- Moved `redmine/plugins` to `plugins` to match official Redmine docker structure.
- Mount postgres volume not as host file.
- Copy official Redmine 4.0 docker's following files:
   - `Dockerfile`: https://github.com/docker-library/redmine/blob/master/4.0/Dockerfile
   - `docker-entrypoint.sh`: https://github.com/docker-library/redmine/blob/master/4.0/docker-entrypoint.sh
- Then modify `Dockerfile` based on @aruneko san's following improvement:
   - https://github.com/aruneko/redmica/blob/dockerize-1.1/Dockerfile
   - But, note that I kept original `Dockerfile` as much as possible.
- Modify `docker-entrypoint.sh`.
   - Replaced only adapter `postgres` part to `postgis`.
   - RGeo itself seems to support `mysql2spatial` and `spatialite` adapter, so I kept original file as much as possible for future work.
- Made configurable `GEM_PG_VERSION` from `docker-compose.yml` to install same `pg` version with Redmine original `Gemfile`.
- Supporting both Redmine/Redmica setup, but currently, only Redmine 4.0 is runnable because of the following Redmine core side issue.
   - [Defect #33121: IssueQuery not usable from plugins - Redmine](https://www.redmine.org/issues/33121)
   - [Patch #33290: Stop DB access when IssueQuery class is defined - Redmine](https://www.redmine.org/issues/33290)
   - You can reproduce above error by modifying `docker-compose.yml` as follows, then execute `docker-compose up`.
      ```diff
      version: "3.4"
      
      x-var: &GEM_PG_VERSION
     -  1.1.4
     +  1.2.2
      
      services:
      
     @@ -10,8 +10,8 @@ services:
            context: ./
            dockerfile: ./Dockerfile
            args:
     -        REDMINE_VERSION: 4.0.7
     -        REDMICA_VERSION: ""
     +        REDMINE_VERSION: ""
     +        REDMICA_VERSION: 1.2.0
              GEM_PG_VERSION: *GEM_PG_VERSION
          ports:
            - 3000:3000
      ```

@gtt-project/maintainer

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "redmine/plugins/redmine_text_blocks"]
```
Comment (sanak) [inline .gitmodules:1]: @dkastl (CC: @aruneko)  
Oh, I noticed that git submodule tag is not updated...  
I will fix this now.

**Review by dkastl — APPROVED** [review]
Thanks!

**Review by aruneko — APPROVED** [review]
LGTM 👍

**Comment by sanak** [comment]
@dkastl @aruneko  
Okay, thanks for confirmation!  
I will merge this now.

**Pull request merged**